.travis.yml

```diff
@@ -86,3 +86,20 @@ after_success:
 
 before_cache:
   - "find ${TRAVIS_BUILD_DIR}/.tox/${TOXENV} -name 'log' -o -name '__pycache__' -type d | xargs -I {} rm -rf {}"
+
+jobs:
+  include:
+  - stage: PyPI Release
+    if: tag IS present
+    # Override script, install, etc. to no-op
+    script: echo "Releasing to PyPI..."
+    install: true
```

Comment:
Buil stages use the test job settings by default, so these need to be overridden so that the installation and test steps don't get run.